Demo/DemoApp/readme.md

```diff
@@ -0,0 +1,81 @@
+# EarlGrey 2 + Carthage
```

Comment:
Nit: I'd capitalize the file name to `README.md`

Reply:
Done!